packages/playback-core/src/tracks.ts

```diff
@@ -137,3 +139,106 @@ export function removeTextTrack(mediaEl: HTMLMediaElement, track: TextTrack) {
   );
   trackElement?.remove();
 }
+
+const DEFAULT_CUEPOINTS_TRACK_LABEL = 'cuepoints';
```

Comment:
thought: should we namespace this like mux-cuepoints? Will help avoid (unlikely) naming collisions.

Reply:
I would leave this as is, `cuepoints`. seems very unlikely to collide